.github/workflows/test.yml

```diff
@@ -149,7 +149,7 @@ jobs:
       - name: "Prepare (network driver=slirp4netns, port driver=builtin)"
         run: DOCKER_BUILDKIT=1 docker build -t ${TEST_TARGET} --target ${TEST_TARGET} --build-arg UBUNTU_VERSION=${UBUNTU_VERSION} --build-arg CONTAINERD_VERSION=${CONTAINERD_VERSION} .
       - name: "Test (network driver=slirp4netns, port driver=builtin)"
-        run: docker run -t --rm --privileged ${TEST_TARGET}
+        run: docker run -t --rm --privileged -e WORKAROUND_CIRRUS=1 ${TEST_TARGET}
```

Comment:
This is not Cirrus, so I guess we have to rename the var to something like WORKAROUND_ISSUE_622

Reply:
I’m also wondering if we can try other workarounds, such as using 8.8.8.8? (From my memory we might have already tried that though)

Reply:
Thanks @AkihiroSuda 

It has been change to the WORKAROUND_ISSUE_622 :-)